test/e2e/router_compression_test.go

```diff
@@ -0,0 +1,261 @@
+// +build e2e
+
+package e2e
+
+import (
+	"context"
+	"crypto/tls"
+	"errors"
+	"fmt"
+	"net/http"
+	"strings"
+	"testing"
+	"time"
+
+	"github.com/openshift/cluster-ingress-operator/pkg/operator/controller"
+	ingress "github.com/openshift/cluster-ingress-operator/pkg/operator/controller/ingress"
+
+	operatorv1 "github.com/openshift/api/operator/v1"
+	routev1 "github.com/openshift/api/route/v1"
+
+	"k8s.io/apimachinery/pkg/types"
+	"k8s.io/apimachinery/pkg/util/wait"
+)
+
+func TestRouterCompressionParsing(t *testing.T) {
+	t.Helper()
```

Comment:
`t.Helper()` should be called in the helper functions, not in the test function itself.

Reply:
👍🏼